README.md

```diff
@@ -227,19 +215,6 @@ Options:
   --help                          Show this message and exit.
 ```
 
-## Performance
-
-On a PC with 32GB memory and Intel Core i7-8850H CPU @ 2.6 ghz, the following performance has been measured:
-
-| Geopackage size | Time needed for validation | MB / minute     |
-| --------------- | -------------------------- | --------------- |
-| 315 MB          | 0.5 minutes                | 630 MB / minute |
-| 6.3 GB          | 12.5 minutes               | 504 MB / minute |
-| 9.9 GB          | 17.5 minutes               | 565 MB / minute |
-| 15.7 GB         | 24 minutes                 | 654 MB / minute |
-
-This is to give an indication of the performance and by no means a guarantee.
```

Comment:
Ik vind dit soort informatie interessant. Echter is de performance afhankelijk van de validaties die uitgevoerd worden, dus akkoord om weg te halen (of anders zou dit uitgebreid en geactualiseerd moeten worden).

Reply:
Op zich eens, maar dan moeten we dit genereren, het is niet actueel meer.

Reply:
Nu geeft het een incorrect beeld.